monitor/charts.py

```diff
@@ -42,13 +43,14 @@ def historical_work_orders_line_chart(data: dict):
 
 def stakers_breakdown_pie_chart(data):
     staker_breakdown = list(data.values())
-    colors = ['#FAE755', '#74C371', '#3E0751']  # colors from Viridis colorscale
+    colors = ['green', 'red', '#e0b32d']  # [active, inactive, pending] (sorted labels)
```

Comment:
maybe these should be done with classes instead of hardcoding the colors into the python code.
then you can style the classes with CSS

Reply:
Ideally yes - but dash is a bit tricky. There may be a way to do it via css, but each segment would need a css id/classname, and then set matching css style. However, through dash, I don't know how to set the id/classname for segments of the pie chart. We can set the id/classname for the pie chart itself, but I'm not sure about individual segments. You can try using Chrome to inspect the pie chart element to see what I mean.

This was part of the styling issues for the Dash DataTable and all of the `!important;` entries in its css 😅 .

Unsure if @KPrasch has seen a way to do this...?

Reply:
ah that's... interesting.  I appreciate your efforts on this more now.  That sounds a bit tedious!